components/plurals/src/rules/ast.rs

```diff
@@ -264,6 +264,11 @@ pub enum Operand {
     F,
     /// Visible fraction digits without trailing zeros
     T,
+    /// Compact decimal exponent value:
+    ///   exponent of the power of 10 used in compact decimal formatting
+    C,
+    /// Currently, synonym for ‘c’. however, may be redefined in the future
+    E,
```

Comment:
I think I would only have C here.  I think `e` should only be used when parsing.